docs/users.md

```diff
@@ -45,9 +45,15 @@ rs0:PRIMARY> db.test.findOne()
 To automate the deployment and management of the cluster components, 
 the Operator requires system-level MongoDB users.
 
-During installation, the Operator requires Kubernetes Secrets to be deployed
-before the Operator is started. The name of the required secrets can be set in
+Credentials for these users are stored as a [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) object.
+The Operator requires Kubernetes Secrets before the database cluster is
+started. It will either use existing Secrets or create a new Secrets object with
+randomly generated passwords if it didn’t exist.
```

Comment:
`if it didn’t exist` -> `if they didn’t exist` (that is if we stick to using plural for Secrets)
But it seems to me it would be more meaningful to just convert all this to singular Secret